src/app/App.vue

```diff
@@ -92,16 +92,28 @@ const shouldShowOnboardingNotification = computed(() => store.getters.shouldShow
 const shouldShowBreadcrumbs = computed(() => store.getters.shouldShowBreadcrumbs)
 
 watch(() => isWizard.value, setIsWizardPageClass, { immediate: true })
+watch(() => [route.meta.title, route.meta.getTitle, store.state.policyTypesByPath], setDocumentTitle, { immediate: true })
 
-watch(() => route.meta.title, function (pageTitle) {
-  setDocumentTitle(pageTitle)
-})
+// Fetches policy types as they’re used to set document titles.
+fetchPolicyTypes()
```

Comment:
No, not for all, but it’s quite convenient to make this call once in App.vue (or maybe a different place in the future) because it’s a small response size and we make use of it in a couple of views:

- all policy views
- DPP policies tab
- mesh detail view